controllers/feeds/getFeedsV2.js

```diff
@@ -144,6 +145,7 @@ module.exports = async (req, res) => {
         const paramGetFeeds = {
           limit: getstreamLimit,
           reactions: {own: true, recent: true, counts: true},
+          ranking: GETSTREAM_RANKING_METHOD,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `ranking` parameter to the `paramGetFeeds` object is a good way to introduce a ranking method for retrieving feeds. However, it's important to ensure that the `GETSTREAM_RANKING_METHOD` constant is defined and valid for the GetStream API you're using. If it's not defined or invalid, it could lead to unexpected behavior or errors when fetching feeds.
```diff
+           ranking: GETSTREAM_RANKING_METHOD ? GETSTREAM_RANKING_METHOD : 'default',
```
This change ensures that if `GETSTREAM_RANKING_METHOD` is not defined, the default ranking method will be used.


<!-- This is an auto-generated comment by OSS CodeRabbit -->